index.html

```diff
@@ -81,15 +81,15 @@
             <div class="col-lg-4 col-lg-offset-2 text-center contact-buttons">
                 <p>Interested in the Research Bazaar? We want to hear from you! Email us your questions, start a conversation about bringing ResBaz to your home town, or just say hi; you can also always find us on Twitter.</p>
 
-                <a href="#" class="btn btn-lg btn-outline light-bkg">
+                <a href=""mailto:research.bazaar@gmail.com" class="btn btn-lg btn-outline light-bkg">
                     <i class="fa fa-envelope"></i> Email
                 </a>
-                <a href="https://twitter.com/resplat" class="btn btn-lg btn-outline light-bkg">
+                <a href="https://twitter.com/resbaz" class="btn btn-lg btn-outline light-bkg">
                     <i class="fa fa-twitter"></i> Twitter
                 </a>
             </div>                    
             <div class="col-lg-4 text-center">
-              <a class="twitter-timeline"  href="https://twitter.com/ResPlat" data-widget-id="652942264121950208">Tweets by @ResPlat</a>
```

Comment:
Not sure this will work due to the unique widget id - see https://twitter.com/settings/widgets/new/user